Bottom: 6fb99b94d9f16c9d04c4593bd468a52e5be033ae
Top:    9baa824ae500ce34bf26ab1970aef976ceb0687a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-04 15:39:44 +0000

Refresh of crosshair-c-when-drawing-via-d

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 58a245e..a8f30f4 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -594,7 +594,7 @@ DrawAttached (Boolean BlockToo)
 	  s = Settings.ViaThickness / 2 + PCB->Bloat + 1;
 	  gui->set_color (Crosshair.GC, Settings.CrossColor);
 	  gui->draw_arc (Crosshair.GC,
-			 Crosshair.X, Crosshair.Y, s, s, 0, 360 * 64);
+			 Crosshair.X, Crosshair.Y, s, s, 0, 360);
 	  gui->set_color (Crosshair.GC, Settings.CrosshairColor);
 	}
       break;
